test/reflection.jl

```diff
@@ -364,9 +364,7 @@ function test_typed_ast_printing(f::ANY, types::ANY, must_used_vars)
     for str in (sprint(io->code_warntype(io, f, types)),
                 sprint(io->show(io, li)))
         # Test to make sure the clearing of file path below works
-        # If we don't store the full path in line number node/ast printing
-        # anymore, the test and the string replace below should be fixed.
-        @test contains(str, @__FILE__)
+        @test string(li.def.file) == @__FILE__
```

Comment:
Why did this test change but not the line after it (which is what this test and the deleted comment were here to protect)